ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -304,8 +304,9 @@ export default class ExperimentalTab extends PureComponent {
                 >
                   {t('addSnapAccountToggle')}
                 </Text>
-                <Box data-testid="add-snap-account-toggle">
+                <Box>
```

Comment:
Is the box still needed? I think I added it just to use the `data-testid`

Reply:
We can remove it.

removed in this commit 580f7138f9641dc678515a831fd15fa167773d54